atd-etl/cris_import/create_fact_table_public_views.py

```diff
@@ -0,0 +1,584 @@
+        "bastion_host": {
+            "opitem": "RDS Bastion Host",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Host",
+            "opvault": VAULT_ID,
+        },
+        "bastion_ssh_username": {
+            "opitem": "RDS Bastion Host",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.ssh Username",
+            "opvault": VAULT_ID,
+        },
+        "database_host": {
+            "opitem": "Vision Zero Database",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Host",
+            "opvault": VAULT_ID,
+        },
+        "database_username": {
+            "opitem": "Vision Zero Database",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Username",
+            "opvault": VAULT_ID,
+        },
+        "database_password": {
+            "opitem": "Vision Zero Database",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Password",
+            "opvault": VAULT_ID,
+        },
+        "database_name": {
+            "opitem": "Vision Zero Database",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Name",
+            "opvault": VAULT_ID,
+        },
+        "database_ssl_policy": {
+            "opitem": "Vision Zero Database",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database SSL Policy",
+            "opvault": VAULT_ID,
+        },
+        "aws_access_key": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Access key",
+            "opvault": VAULT_ID,
+        },
+        "aws_secret_key": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Secret key",
+            "opvault": VAULT_ID,
+        },
+        "s3_archive_bucket_name": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.S3 Archive Bucket Name",
+            "opvault": VAULT_ID,
+        },
+        "s3_archive_path": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.S3 Archive Path",
+            "opvault": VAULT_ID,
+        },
+        "sftp_endpoint_private_key": {
+            "opitem": "SFTP Endpoint Key",
+            "opfield": ".private key",
+            "opvault": VAULT_ID,
+            },
+        "bastion_ssh_private_key": {
+            "opitem": "RDS Bastion Key",
+            "opfield": ".private key",
+            "opvault": VAULT_ID,
+            },
+    }
+
+    # instantiate a 1Password client
+    client: Client = new_client(ONEPASSWORD_CONNECT_HOST, ONEPASSWORD_CONNECT_TOKEN)
+    # get the requested secrets from 1Password
+    return onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
+
+
+def make_crashes_view():
+    pg = get_pg_connection()
+    db = pg.cursor(cursor_factory=psycopg2.extras.DictCursor)
+    
+    db.execute("drop view if exists public.atd_txdot_crashes;")
+    pg.commit()
+
+    # ldm.atd_txdot_crashes columns we want to compute on the fly
```

Comment:
Could you expand on this comment some? Might be an artifact from previous iterations.

Reply:
I've removed that comment. If anything, it's more detritus left over from trying to figure out what to do with what fields, and I think [my previous comment](https://github.com/cityofaustin/atd-vz-data/pull/1288#discussion_r1337797768) outlines my preferred way to handle getting all the fields correct.